index.js

```diff
@@ -24,67 +25,84 @@ function getRandomNumber(min, max) {
 
 function checkGuess() {
   // Get value from guess input element
-  const guess = parseInt(guessInput.value, 10);
-  attempts = attempts + 1;
-
+  let guess = parseInt(guessInput.value, 10);
+  console.log(guess)
+  
   hideAllMessages();
 
+
   if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
+
+    
     numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
+    numberOfGuessesMessage.style.display ='';
 
-    correctMessage.style.display = '';
 
     submitButton.disabled = true;
     guessInput.disabled = true;
   }
 
-  if (guess !== targetNumber) {
-    if (guess < targetNumber) {
-      tooLowMessage.style.display = '';
-    } else {
-      tooLowMessage.style.display = '';
-    }
+  if (guess < targetNumber) {
+    tooLowMessage.style.display = '';
+  }
+  if (guess > targetNumber) {
+    tooHighMessage.style.display = '';
+  }
 
-    const remainingAttempts = maxNumberOfAttempts - attempts;
+  {
 
-    numberOfGuessesMessage.style.display = '';
-    numberOfGuessesMessage.innerHTML = `You guessed ${guess}. <br> ${remainingAttempts} guesses remaining`;
-  }
 
-  if (attempts ==== maxNumberOfAttempts) {
-    submitButton.disabled = true;
-    guessInput.disabled = true;
+
+
+
+    const remainingAttempts = maxNumberOfAttempts - attempts;
+    console.log(remainingAttempts)
```

Comment:
the value of maximum attempts is 5. i changed the value of attempts to 1. now it shows guess remaining 4. but i notice the number 4 doesn't go down when i change the guess number.

Reply:
That's right :+1:  so right now what you have is that `maxNumberOfAttempts` is always 5; that makes sense since we don't want to change the total number of guesses that the player is allowed to make.

And you're right, that once you change attempts to 1, now there's only going to be 4 remaining guesses. Where can we write that update to the `attempts` variable so that it changes (adds one more attempt) each time the user **submits** a guess?

Reply:
I added attempts++ outside the for loop.

Reply:
I changed this & added attempts++ under maxGuessesMessage.innerHTML = `You guessed ${guess} <br>   guesses remaining ${remainingAttempts}`;
  maxGuessesMessage.style.display = '';

but still getting negative numbers